gcloud-java-dns/src/main/java/com/google/gcloud/dns/DnsImpl.java

```diff
@@ -110,6 +110,16 @@
     dnsRpc = options.rpc();
   }
 
+  static Function<ManagedZone, Zone> pbToZoneFunction(final DnsOptions options) {
```

Comment:
Nit: from the other modules we have so far, we tend to say something like "fromPb", or in this case because it's zone-specific, maybe "zoneFromPb".  It's not necessary, but it'd be nice to keep that convention so that it's clearly differentiated from other modules' "toPb" functions.